packages/migrations/src/actions/2025.10.17T00-00-00.project-access-tokens.ts

```diff
@@ -0,0 +1,37 @@
+import { type MigrationExecutor } from '../pg-migrator';
+
+export default {
+  name: '2025.10.17T00-00-00.organization-access-tokens-project-scope.ts',
+  noTransaction: true,
+  run: ({ sql }) => [
+    {
+      name: 'add new columns to "organization_access_tokens"',
+      query: sql`
+        ALTER TABLE "organization_access_tokens"
+          ADD COLUMN IF NOT EXISTS "project_id" UUID REFERENCES "projects" ("id") ON DELETE CASCADE
```

Comment:
If this column is populated, the access token is a access token bound to a project scope.

Reply:
<img width="1822" height="941" alt="image" src="https://github.com/user-attachments/assets/f225536b-add9-4efe-bccb-150fe0da9000" />

Reply:
A filter should work in that case?